### ISSUE TITLE
padding-around-expect-groups doesn't detect `expectTypeOf` as an expect clause

### ISSUE THREAD

**Issue by eranhirsch** [vitest-dev/eslint-plugin-vitest#585]
The rule `vitest/padding-around-expect-groups` isn't detecting the following cases:

```ts
test("my test", () => {
  const result = func(...);
  expectTypeOf(result).toEqualTypeOf<...>();
});
```

but works fine with the following cases:

```ts
test("my test", () => {
  const result = func(...);
  expect(result).toEqual(...);
});
```
